frontend/spec/selenium/common/webdriver.rb

```diff
@@ -2,8 +2,6 @@
 
 require 'selenium-webdriver'
 
-require 'pry'
```

Comment:
@quoideneuf I think we've got to keep this put, or at least leave it until we've come up with another plan to make pry available.  I know many folks (myself included) are in the habit of throwing in some `binding.pry`s when debugging the tests, not to mention the `ASPACE_TEST_WITH_PRY` environment variable that pops up several places in the selenium tests.  Thoughts?

Reply:
@lorawoodford I reverted that, did not mean to disrupt anything.